ckanext/data_qld/de_identified_data/helpers.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from ckan import authz
+
+log = logging.getLogger(__name__)
+
+
+def process_de_identified_data_dict(data_dict, user_obj):
+    """
+    Remove de_identified_data value from dict
+    if current user doesn't have access to it.
+    """
+    # Add default value to NO.
+    if data_dict.get('de_identified_data', None) is None:
+        data_dict['de_identified_data'] = 'NO'
+
+    if not show_de_identified_data(data_dict, user_obj):
+        del data_dict['de_identified_data']
+
+
+def user_has_adm_editor_org_access(group_id, user_id):
```

Comment:
Can you please spell the whole word admin instead of adm in method name

Reply:
updated.